src/myorganization/EditOrganization.jsx

```diff
@@ -81,201 +85,210 @@ export default function EditOrganization() {
+                  maxLength="20"
+                  pattern="^[0-9\s\-\+]+$"
+                  onChange={validatePhone}
+                />
+              </p>
+              {!isValidPhone && (
+                <span className="error">
+                  Must only contain numbers between 0-9, may start with plus "+" before country code, and may have dash "-"
+                </span>
+              )}
+            </form-group>
+            <div><br></br></div>
+            <div className="input-group mb-3">
+              <p className="input-control">
+                <label htmlFor="status">Status</label>
+                <select className="custom-select" name="status" id="status">
+                  <option
+                    defaultValue="Draft">{organization.status}
+                  </option>
+                  {ORGANIZATION_STATUS.map((status) => optionsWithDefaultSelection(status, organization.status))}
+                </select>
+              </p>
+            </div>
+            <div><br></br></div>
+            <form-group controlId="formJoinDate">
+              <p className="input-control">
+                <label id="joinDate">Join date :</label>
+                <input className="field"
+                  type="url"
+                  aria-labelledby="joinDate"
+                  name="joinDate"
+                  defaultValue={organization.join_date}
+                  disabled
+                />
+              </p>
+            </form-group>
+            <div><br></br></div>
+            <div><br></br></div>
+            <div className="row justify-content-between">
+              {responseStatus === 200 &&
+
                 <div className="col-sm-6">
                   <Link to={{
-                      pathname: `/organization-portfolio`,
-                      state: { organization }
-                      }}
+                    pathname: `/organization-portfolio`,
+                    state: { organization }
+                  }}
                     className="btn btn-primary"
                     variant="primary"
                     name="toPrograms"
                     organization={organization}
                   >Go to Programs
                   </Link>
                 </div>
-                <div>
+
+
+              }
+
+              <div>
                 {responseMessage &&
-                <span className="error" name="response" aria-label="response" role="alert">{responseMessage}</span>}
+                  <span className="error" name="response" aria-label="response" role="alert">{responseMessage}</span>}
               </div>
-              <div className="row">
-                <div className="col-sm-6">
-                  <button className="btn btn-success"
-                          variant="success"
-                          type="submit"
-                          name="submit"
-                          value="Save"
-                  >Save
+              {responseStatus === 200 &&
```

Comment:
You can make a separate row for the 2 buttons (`Go to Programs` and `Save`) below the responseMessage. This way we can just do one responseStatus check in the code.

Reply:
I did that but it was interfering with the CSS , since that's not an issue I'll make the change

Reply:
i'll make the PR on a new branch ASAP